aws-janitor/resources/target_groups.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
It's copied from [elbv2.go](https://github.com/kubernetes-sigs/boskos/blob/3416530a0ed99825d6a74764ad57e60f98d5e0e7/aws-janitor/resources/elbv2.go), but I'm not 100% sure whether some logics for load balancer cleanup should also be implemented for target groups